platform/web/detect.py

```diff
@@ -237,30 +240,20 @@ def configure(env: "SConsEnvironment"):
         env["proxy_to_pthread"] = False
 
     if env["lto"] != "none":
-        # Workaround https://github.com/emscripten-core/emscripten/issues/19781.
-        if cc_semver >= (3, 1, 42) and cc_semver < (3, 1, 46):
-            env.Append(LINKFLAGS=["-Wl,-u,scalbnf"])
-        # Workaround https://github.com/emscripten-core/emscripten/issues/16836.
-        if cc_semver >= (3, 1, 47):
-            env.Append(LINKFLAGS=["-Wl,-u,_emscripten_run_callback_on_thread"])
+        env.Append(LINKFLAGS=["-Wl,-u,_emscripten_run_callback_on_thread"])
```

Comment:
```suggestion
        # Workaround https://github.com/emscripten-core/emscripten/issues/16836.
        env.Append(LINKFLAGS=["-Wl,-u,_emscripten_run_callback_on_thread"])
```

Reply:
It doesn't show as "outdated", but I definitely added the comment.